avanza/avanza.py

```diff
@@ -405,6 +405,14 @@ def get_deals_and_orders(self) -> DealsAndOrders:
         """Get currently active deals and orders"""
         return self.__call(HttpMethod.GET, Route.DEALS_AND_ORDERS_PATH.value)
 
+    def get_deals(self):
+        """ Get currently active deals """
+        return self.__call(HttpMethod.GET, Route.DEALS_PATH.value)
+
+    def get_orders(self):
+    """ Get currently active orders """
```

Comment:
This comment is wrongly indented, should be indented to be in line with the code.